erpnext/controllers/subcontracting_controller.py

```diff
@@ -103,6 +102,13 @@ def validate_items(self):
 						_("Row {0}: Item {1} must be a subcontracted item.").format(item.idx, item.item_name)
 					)
 
+				po_item = frappe.get_doc("Purchase Order Item", item.purchase_order_item)
```

Comment:
Use of get_doc in for loop is expensive, better use single db query to get required data for all items